docs/xapi/mapping/problems.md

```diff
@@ -0,0 +1,196 @@
+# Problems
+
+For now, we only focus on server events.
+
+## `problem_check`
+
+### Mapping
+
+|                                xAPI Key                               |                                                    Value                                                    |
+|:---------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------------------:|
+| Actor                                                                 |                                                                                                             |
```

Comment:
Would it be more clear if we separate visually the "title" rows from other rows (e.g. with bold text)? WDYT?